matrix-sdk-android/src/main/java/fr/gouv/tchap/android/sdk/internal/session/users/TchapDefaultUserInfoService.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.gouv.tchap.android.sdk.internal.session.users
+
+import fr.gouv.tchap.android.sdk.api.session.userinfo.TchapUserInfoService
+import fr.gouv.tchap.android.sdk.api.session.userinfo.model.TchapUserInfo
+import javax.inject.Inject
+
+internal class TchapDefaultUserInfoService @Inject constructor(
```

Comment:
Please rename to TchapUserInfoService

Reply:
This causes a conflict between the interface and the implementation names

Reply:
do you have an equivalent in element-android?

Reply:
removed tchap prefix from interfaces